components/cards/ActionForm.tsx

```diff
@@ -265,6 +272,25 @@ export function ActionForm(props: ActionFormProps) {
                 />
               </Form.Group>
               <br />
+              <hr className="action-form_divider" />
+              <br />
+              <ClaimView
+                stream={
+                  networkFeeRatePerSecond
+                    ? truncateEth(formatBalance(networkFeeRatePerSecond), 18)
+                    : "0"
+                }
+                streamBuffer={
+                  networkFeeRatePerSecond
+                    ? truncateEth(
+                        formatBalance(networkFeeRatePerSecond.mul(2 * 60 * 60)),
+                        18
+                      )
+                    : "0"
+                }
```

Comment:
**Update**: the `Stream` and `StreamBuffer` are calculated here.

Reply:
Is the buffer here 2 hours? Will it be 2 hours on testnet and 8 hours on a mainnet? If so, can we make this dynamic based on the network?

Reply:
The constants [here](https://github.com/superfluid-finance/protocol-monorepo/blob/dev/packages/sdk-core/src/constants.ts) may be useful